packages/button/test/button.test.ts

```diff
@@ -74,748 +74,726 @@ describe('Button', () => {
                 },
             });
         });
-        it('warns in devMode when white/black static is provided', async () => {
+
+        it('loads default', async () => {
             const el = await fixture<Button>(html`
-                <sp-button tabindex="0" static="black">Button</sp-button>
+                <sp-button>Button</sp-button>
             `);
 
             await elementUpdated(el);
-            expect(consoleWarnStub.called).to.be.true;
+            expect(el).to.not.be.undefined;
+            expect(el.textContent).to.include('Button');
+            await expect(el).to.be.accessible();
 
-            const spyCall = consoleWarnStub.getCall(0);
-            expect(
-                (spyCall.args.at(0) as string).includes('deprecated'),
-                'confirm deprecated static warning'
-            ).to.be.true;
-            expect(spyCall.args.at(-1), 'confirm `data` shape').to.deep.equal({
-                data: {
-                    localName: 'sp-button',
-                    type: 'api',
-                    level: 'deprecation',
-                },
-            });
+            // Applies a default variant as an stylable attribute
+            expect(el.variant).to.equal('accent');
+            expect(el.getAttribute('variant')).to.equal('accent');
         });
-        it('warns in devMode when white/black static is provided', async () => {
+        it('loads default w/ element content', async () => {
             const el = await fixture<Button>(html`
-                <sp-button tabindex="0" static="white">Button</sp-button>
+                <sp-button label="Button"><svg></svg></sp-button>
```

Comment:
Even a ghost declaration takes some bandwidth. can we remove the redundant svg registration?

Reply:
This whole test is redundant and evergreen. I've removed it altogether.

Reply:
Thanks!